sintr_worker/lib/basic_info.dart

```diff
@@ -87,13 +86,31 @@ String logKey, Stream<String> stream) async {
         return result;
       }
     }
-    if (line.startsWith(data_prefix)) {
-      // Extract the info and add a result of the format
-      // <millisSinceEpoch>:Ver:<uuid>:<clientId>:<clientVersion>:<serverVersion>:<sdkVersion>
-      result[basic_info_key] = [line.substring(data_prefix.length)];
+    if (line.startsWith('Data :: ~')) {
+      int index = line.indexOf(':Ver:');
+      if (index != -1) {
+        int milliSinceEpoch = int.parse(line.substring(9, index));
+        var prefix = key_prefix;
+        var key = buildKey(prefix, milliSinceEpoch);
+        // Extract the info and add a key/value of the format
+        // basic_info_<YYYY>-<MM>-<DD>
+        // <uuid>:<clientId>:<clientVersion>:<serverVersion>:<sdkVersion>
+        result[key] = [line.substring(index + 5)];
+      }
       return result;
     }
   }
-  result[basic_info_key] = [unknown_value];
+  result[key_prefix] = [unknown_value];
   return result;
 }
+
+/// Return a key of the form <prefix><YYYY>-<MM>-<DD>
+String buildKey(String prefix, int milliSinceEpoch) {
```

Comment:
Add a todo to consider something like this: http://stackoverflow.com/questions/16126579/how-do-i-format-a-date-with-dart

when we have the inftastructure in place to do pub-deployments to the worker nodes?